hazelcast/src/main/java/com/hazelcast/sql/impl/client/SqlClientService.java

```diff
@@ -109,6 +113,10 @@ public SqlResult execute(@Nonnull SqlStatement statement) {
         }
     }
 
```

Comment:
Maybe we can add explain motivation the behind this method(or field if we will change to field after Metin's review) shortly. We can state that phone statistics updates are skipped in MC client not to screw up phone home statistics. (as explained in https://github.com/hazelcast/hazelcast/issues/19498)

We can also add a comment to execute() in member side.